Extra/Shaders/NewVegasReloaded/Shaders/SKYTEX.pso.hlsl

```diff
@@ -122,15 +122,15 @@ VS_OUTPUT main(VS_INPUT IN) {
         finalColor.rgb *= color.rgb * Params.y;
         
         finalColor = float4(finalColor.rgb + (isSun * sunColor) * TESR_SunAmount.w, pows(alpha, TESR_SunAmount.w));
-        finalColor.rgb = pows(finalColor.rgb, 1.0/2.2);
+        finalColor.rgb = pows(finalColor.rgb, 1.0/2.2); //delinearise
 
         OUT.color_0 = finalColor;
 
         return OUT;
     }
 
     // shade clouds 
-    float greyScale = lerp(luma(finalColor), 1.0, saturate(TESR_CloudData.w));
+    float greyScale = lerp(luma(finalColor), 1.0, saturate(TESR_CloudData.w * TESR_DebugVar.w));
```

Comment:
it would be good to remove the debug vars from the shaders before release

Reply:
Also removed